bedrock/firefox/views.py

```diff
@@ -568,7 +555,9 @@ def new(request):
             elif experience == 'batmresist':
                 template = 'firefox/new/batm/resist.html'
             else:
-                template = 'firefox/new/scene1.html'
+                template = 'firefox/new/onboarding/scene1.html'
+        elif lang_file_is_active('firefox/new/onboarding', locale):
```

Comment:
@flodolo a third one here (but can probably share the same lang file as `'firefox/new/onboarding/scene2.html'`